packages/testing-kit/tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Please extend from the base file in the root, I guess most rule won't be needed then